pom.xml

```diff
@@ -424,22 +425,45 @@
 
                 </executions>
             </plugin>
+            <plugin>
+                <artifactId>maven-failsafe-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>integration-test</id>
+                        <goals>
+                            <goal>integration-test</goal>
+                            <goal>verify</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
         </plugins>
 
     </build>
     <profiles>
         <profile>
-            <id>testsuite</id>
+            <id>excludeTests</id>
```

Comment:
What's the use case of this profile ?

Reply:
To exclude any test, it's used in jenkins for example (not sure if still needed) I left it as it was just moved it so remote test exclusion is actually working

Reply:
Don't know about this use case but in general if the test is failing it's better to ignore it on the test source level.
If it's special type of test that needs to be excluded by default an option is to create a "group" and exclude it.

It's good to have a project in s state that it builds and all the tests pass for anyone with standard `mvn clean install` command.

Reply:
I have no problem to ignore class that's excluded in jenkins and remove this profile, but as I said I don't know reasoning behind this approach